docs/04-reference/compatibility/compatibility.json

```diff
@@ -1,669 +1,1751 @@
 {
```

Comment:
Can we undo the formatting changes? (it makes it harder to review the changes) We should handle this in a separate PR and add a dedicated lint command / build step

Reply:
donoe, and adding queue.pop too